docs/plugins/image.md

```diff
@@ -1,77 +1,81 @@
 # Image
 
-The standard way of displaying images in Lightning is to just specify the `src`. This is the prefered way for your App's local assets (such as background, splash screen, logo and icons).
+The standard way of displaying images in Lightning is to just specify the `src`. This is the preferred way for your local assets (such as background, splash screen, logo and icons) of Lightning Apps.
 
-It's adviced that you optimize an App's local assets, by resizing them to the exact size and quality you will be using them. This will be beneficial for the memory usage of your App.
+It is recommended that you *optimize* the local assets of your App by resizing them to the *exact* size and quality in which you will use them. This positively affects the memory usage of your App.
 
-However when you don't have control over the images you're displaying in your App (because they come from a remote API for example), you can use the Image plugin to resize and crop images.
+However, if you don't have control over the images to be displayed in your App (for example, because they originate from a remote API), you can use the *Image* plugin to resize and crop them.
 
 ## Usage
 
-Whenever you need to resize image, import the Image plugin from the Lightning SDK
+Import the Image plugin from the Lightning SDK in components where you want to resize an image:
 
 ```js
 import { Img } from '@lightningjs/sdk'
 ```
 
-## Available methods
+## Available Methods
 
-### Exact
+### exact()
```

Comment:
I'm not sure if I like this change ... (multiple occurrences, so only mentioning once)
Let's discuss

Reply:
Agreed to remove the parentheses (see discusion in Skype chat of February 23rd).